### ISSUE TITLE
SBT 1.3. and junit-interface 0.11 not running JUnit tests

### ISSUE THREAD

**Issue by njlbenn** [sbt/junit-interface#93]
This is just meant to be a duplicate of https://github.com/sbt/junit-interface/issues/35 but for SBT 1.3, and putting SBT 1.3 in the title of the issue to help anyone searching on the web for a solution.

I came across https://github.com/sbt/junit-interface/issues/35 while trying to understand why my JUnit tests stopped running when I moved from SBT 1.2.8 to 1.3.13 (both with junit-interface 0.11).

Basically the same `crossPaths := false` solution from https://github.com/sbt/junit-interface/issues/35 works in the case of SBT 1.3.

**Comment by unkarjedy** [comment]
we use SBT 1.3.13 and tests work fine without any workarounds
